### ISSUE TITLE
about Loss function

### ISSUE THREAD

**Issue by taherimehran** [Sina-Mohammadi/DFNet#3]
Peace be upon you
I'm interested in using your new cost function in what I'm doing. And I want to use your original code, you may send the pytorch code of your cost function to my email.
Thank you
emial: mehrantaheri369@gmail.com

**Comment by Sina-Mohammadi** [comment]
Hi Mehran,

Thank you for your interest in our paper. As I told you before, we have not
implemented the loss function in Pytorch. Furthermore, I am not familiar
enough with Pytorch to implement the loss function now. However, to
implement my next project, I will definitely learn and use PyTorch. So I
might implement the loss function in Pytorch in 3 months. Sorry if it will
be too late for you.

Best,
Sina
